drivers/docker/driver_test.go

```diff
@@ -378,6 +378,61 @@ func TestDockerDriver_Start_StoppedContainer(t *testing.T) {
 	require.NoError(t, d.DestroyTask(task.ID, true))
 }
 
+// TestDockerDriver_ContainerAlreadyExists asserts that when Nomad tries to start
+// a job and the container already exists, it purges it, and starts it again (as
+// opposed to trying to continuously re-create an already existing container)
+//
+// See https://github.com/hashicorp/nomad/issues/22218
```

Comment:
I'd say remove this link and focus on detailing what the test covers in the comments. The issue might get deleted or similar, which makes it a pointer to maintain over time.

It might be useful to clarify why we purge it, if it already exists.

Reply:
updated the comment in c13d3d0